src/main/java/org/isf/medicalinventory/gui/InventoryEdit.java

```diff
@@ -921,6 +954,32 @@ private JButton getCloseButton() {
 		return closeButton;
 	}
 
+	private JButton getPrintButton() {
+		printButton = new JButton(MessageBundle.getMessage("angal.common.print.btn"));
+		printButton.setMnemonic(MessageBundle.getMnemonic("angal.common.print.btn.key"));
+		printButton.setEnabled(true);
+
+		printButton.addActionListener(e -> {
+			if (inventory == null || inventory.getId() == 0 || inventory.getId() <= 0) {
+				MessageDialog.info(this, "angal.inventory.pleasesavebeforprinting");
+				return;
+			}
+
+			if (!InventoryStatus.done.name().equals(inventory.getStatus())) {
+				int printQtyReal = 0;
+				int response = MessageDialog.yesNo(this, "angal.inventory.askforrealquantityempty");
+				if (response == JOptionPane.YES_OPTION) {
+					printQtyReal = 1;
+				}
+				new GenericReportPharmaceuticalInventory(inventory, "Inventory", printQtyReal);
```

Comment:
Why isn't `"Inventroy"` localized?

Reply:
Inventory is passed when we open InventoryEdit in update mode or view mode